### PR TITLE
Gulag gets a pacman generator

### PR DESCRIPTION
**Pull request by patpol4**
### What is the pr
Adds a pacman to gulag maint 
image

### Why is this change good for the game?
Sometimes the power runs out in there, gives sec a way to recharge provided they have plasma sheets.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
There is a power generator in gulag maint

### What should players be aware of when it comes to the changes your PR is implementing?
There is a power generator in gulag maint

# Changelog

:cl:  
tweak: adds a power generator in gulag maint
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
Wheres the plasma to power it?

**Comment by patpol4** [comment]
> Wheres the plasma to power it?

Well it's a mining station

**Comment by alexkar598** [comment]
Nobody is gonna bother mining to power the gulag while also carrying a prisonier behind them

**Comment by patpol4** [comment]
Right, I'll add like 10 sheets or so

**Comment by patpol4** [comment]
Added 25.
Merge https://github.com/yogstation13/Yogstation/pull/11259 first

**Comment by Cartlord** [comment]
...Why delay the merge, though?

**Comment by patpol4** [comment]
The other pr might get a conflict because it's modifying the same file.

**Review by JamieD1 — APPROVED** [review]
